testing/ncids-js-testing/cypress/integration/InnerPage.feature

```diff
@@ -0,0 +1,8 @@
+Feature: Inner page template
+
+	I want to open see the example inner page
+
+	Scenario: Opening a page template
+		Given I open Example Inner Page template
+		Then I see "Example Inner Page" in the title
```

Comment:
I would suggest rephrasing it as follows since you are using cy.title() command for this step and you have another step below for the page title verification, so it's better from the user perspective to differentiate two titles
Then browser title is "Example Inner Page"